tests/admin/controller/test_controller.py

```diff
@@ -3475,446 +3475,6 @@ def test_admin_auth_service_delete(self):
-            eq_(self._default_library.short_name, library.get("short_name"))
-
-    def _common_basic_auth_arguments(self):
-        """We're not really testing these arguments, but a value for them
-        is required for all Basic Auth type integrations.
-        """
-        B = BasicAuthenticationProvider
-        return [
-            (B.TEST_IDENTIFIER, "user"),
-            (B.TEST_PASSWORD, "pass"),
-            (B.IDENTIFIER_KEYBOARD, B.DEFAULT_KEYBOARD),
-            (B.PASSWORD_KEYBOARD, B.DEFAULT_KEYBOARD),
-            (B.IDENTIFIER_BARCODE_FORMAT, B.BARCODE_FORMAT_CODABAR),
-        ]
-
-    def test_patron_auth_services_post_errors(self):
-        with self.request_context_with_admin("/", method="POST"):
-            flask.request.form = MultiDict([
-                ("protocol", "Unknown"),
-            ])
-            response = self.manager.admin_settings_controller.patron_auth_services()
-            eq_(response, UNKNOWN_PROTOCOL)
-
-        with self.request_context_with_admin("/", method="POST"):
-            flask.request.form = MultiDict([])
-            response = self.manager.admin_settings_controller.patron_auth_services()
-            eq_(response, NO_PROTOCOL_FOR_NEW_SERVICE)
-
-        with self.request_context_with_admin("/", method="POST"):
-            flask.request.form = MultiDict([
-                ("id", "123"),
-            ])
-            response = self.manager.admin_settings_controller.patron_auth_services()
-            eq_(response, MISSING_SERVICE)
-
-        auth_service, ignore = create(
-            self._db, ExternalIntegration,
-            protocol=SimpleAuthenticationProvider.__module__,
-            goal=ExternalIntegration.PATRON_AUTH_GOAL,
-            name="name",
-        )
-
-        with self.request_context_with_admin("/", method="POST"):
-            flask.request.form = MultiDict([
-                ("id", auth_service.id),
-                ("protocol", SIP2AuthenticationProvider.__module__),
-            ])
-            response = self.manager.admin_settings_controller.patron_auth_services()
-            eq_(response, CANNOT_CHANGE_PROTOCOL)
-
-        with self.request_context_with_admin("/", method="POST"):
-            flask.request.form = MultiDict([
-                ("name", auth_service.name),
-                ("protocol", SIP2AuthenticationProvider.__module__),
-            ])
-            response = self.manager.admin_settings_controller.patron_auth_services()
-            eq_(response, INTEGRATION_NAME_ALREADY_IN_USE)
-
-        auth_service, ignore = create(
-            self._db, ExternalIntegration,
-            protocol=MilleniumPatronAPI.__module__,
-            goal=ExternalIntegration.PATRON_AUTH_GOAL
-        )
-
-        common_args = self._common_basic_auth_arguments()
-        with self.request_context_with_admin("/", method="POST"):
-            M = MilleniumPatronAPI
-            flask.request.form = MultiDict([
-                ("name", "some auth name"),
```

Comment:
This isn't present in the new file.

Reply:
We think Edwin added this just to be tidy. We propagated it to its new location.